docker-compose.yml

```diff
@@ -159,12 +171,19 @@ services:
   postgres:
     image: "postgres:16.0-alpine3.18"
     environment:
-      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_USER=${PG_DB_USER:?err}
+      - POSTGRES_PASSWORD=${PG_DB_PASS:?err}
+      - POSTGRES_DB=${PG_DB_NAME:?err}
       - PGDATA=/var/lib/postgresql/data/pgdata
     ports:
       - '5432:5432'
     volumes:
       - *pgsql-volume-data
+    healthcheck:
```

Comment:
Has this been validated @hboutemy ?

Reply:
I got that from postgres official image doc, I tested, I got error message for user root that made me add the `-U`
is there any other validation step your are thinking about?

Reply:
No - I just recall that whatever was the official documented approach to this when I originally added did not work.

If you're happy this works, GREAT!

Reply:
in fact, the interesting side of it is in IQ Server's access configuration
```
    depends_on:
      postgres:
        condition: service_healthy
```

I felt that before this check, there were race conditions (db used before it is ready) that don't happen after
But I can't really prove it strictly

Reply:
tested with the command added in PR description: this really permits to start using the new db only after it is ready
it really does what it is supposed to